src/fsharp/service/service.fs

```diff
@@ -1122,7 +1134,13 @@ type TypeCheckInfo
             (fun () -> 
                 match GetDeclItemsForNamesAtPosition(ctok, None,namesOpt,None,None,line,lineStr,colAtEndOfNames,ResolveTypeNamesToCtors,ResolveOverloads.No,(fun() -> []),fun _ -> false) with
                 | None -> FSharpMethodGroup("",[| |])
-                | Some (items, denv, _, m) -> FSharpMethodGroup.Create(infoReader, m, denv, items |> List.map (fun x -> x.ItemWithInst)))
+                | Some (items, denv, _, m) -> 
+                    let ctors = items |> List.filter (fun x -> match x.Item with Item.CtorGroup _ -> true | _ -> false)
```

Comment:
Could you add a comment about what this is doing - why are we preferring constructors over all other possible items?  What are the other items we are trying to avoid here?  Should we explicitly filter for those?